pkgs/by-name/ne/neovide/package.nix

```
@@ -114,6 +114,6 @@ rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
     changelog = "https://github.com/neovide/neovide/releases/tag/${version}";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ ck3d ];
-    platforms = platforms.all;
+    platforms = platforms.linux ++ [ "aarch64-darwin" ];
```

```suggestion
    platforms = platforms.all;
    broken = stdenv.isDarwin && stdenv.isx86_64;
```

If upstream supports x86_64-darwin.

---

Also am seeing it broke on aarch64-darwin.. getting a failed test. https://gist.github.com/khaneliman/19234e4a5a7e1e7e6aef5a61bb850de9